components/src/core/components/CardTable/Table/TableHeaderSortDropdown.vue

```diff
@@ -60,10 +60,13 @@ import Text from '@orangehrm/oxd/core/components/Text/Text.vue';
 import Icon from '@orangehrm/oxd/core/components/Icon/Icon.vue';
 import IconButton from '@orangehrm/oxd/core/components/Button/Icon.vue';
 import clickOutsideDirective from '../../../../directives/click-outside';
+import translateMixin from '../../../../mixins/translate';
 
 export default defineComponent({
   name: 'oxd-card-th-sort',
 
+  mixins: [translateMixin],
```

Comment:
Yes, we have to use this mixing in future instead $t due the storybook rendering issue. Better to discuss in the dev meeting in the morning

Reply:
used "useTranslate" composable by [deconstructing and renaming as t](https://github.com/orangehrm/oxd/pull/302/files#diff-19d2ac0d702cfb86bdf7ed23cc4549cba0e9ba223f847f25a85b4d9ce63dfb90R69) inside the setup function of "DefaultCardHeader" component because composables can be only used in composition API